indexing-service/src/main/java/io/druid/indexing/worker/WorkerTaskMonitor.java

```diff
@@ -125,10 +124,13 @@ public void run()
         log.info("Started WorkerTaskMonitor.");
         started = true;
       }
+      catch (InterruptedException e) {
+        throw e;
```

Comment:
The convention in most of the java stdlib is that the interrupt flag is cleared when InterruptedException is thrown. I don't see it buying us much to do both... it should be enough to do one or the other.